.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     parallelism: 1
     shell: /bin/bash --login
     docker:
-      - image: circleci/ruby:3.0.3-node
+      - image: cimg/ruby:3.1.0-node
```

Comment:
I changed this from `circleci` to `cimg` because `cimg` is the one we're supposed to use: 

>This image is designed to supercede the legacy CircleCI Ruby image, circleci/ruby.

(From https://circleci.com/developer/images/image/cimg/ruby)